portfolio/src/main/java/com/google/sps/data/CommentPersistHelper.java

```diff
@@ -76,32 +77,45 @@ public void deleteComment(long id) {
     }
   }
 
-  /** Stringifies the comments in the desired order, including pagination */
-  public String stringifyComments(int numberComments, SortMethod sort, int paginationFrom) {
```

Comment:
Something to consider for the future, but not important for now, given that this project is meant as an introduction. 

As we can see from the implementation, stringify() is a complicated operation. I could see another class owning this responsibility.